static/js/components/LeaveProgramWizard.js

```diff
@@ -0,0 +1,171 @@
+// @flow
+/* global SETTINGS: false */
+import React from "react"
+import type { Dispatch } from "redux"
+import { Card } from "react-mdl/lib/Card"
+import Dialog from "material-ui/Dialog"
+import SelectField from "material-ui/SelectField"
+import MenuItem from "material-ui/MenuItem"
+import R from "ramda"
+import { dialogActions } from "./inputs/util"
+import _ from "lodash"
+
+import ProfileFormFields from "../util/ProfileFormFields"
+import type { Profile } from "../flow/profileTypes"
+import type {
+  AvailableProgramsState,
+  AvailableProgram
+} from "../flow/enrollmentTypes"
+import type { UIState } from "../reducers/ui"
+import { showDialog, hideDialog, setProgramsToUnEnroll } from "../actions/ui"
```

Comment:
Minor nitpick, can you move this import and the one below it above the type imports and below the `ProfileFormFields` import?
(https://github.com/mitodl/eslint-config#import-order)